### DISCUSSION TITLE
Add Powered By Page

### DISCUSSION THREAD

**Discussion by yaooqinn** [apache/kyuubi#436] [General]
Make our work count and encourage people to share their brief stories with Kyuubi.
For example, http://spark.apache.org/powered-by.html

I guess this is the right way to better marketing.

cc @pan3793 @Honglun @turboFei @MyLanPangzi @ulysses-you

**Comment by yangrong688** [comment]
I think we can also refer to [who uses apache airflow](https://github.com/apache/airflow#who-uses-apache-airflow) page, to let people make a pr with their company names and @githubhandle.
